packages/tests/src/web-test-runner.config.ts

```diff
@@ -226,7 +226,7 @@ const config: TestRunnerConfig = {
   // enough so that blocked tests have time to wait for all previous test files
   // to run to completion.
   testsStartTimeout: 60000 * 10, // default 120000
-  testsFinishTimeout: 120000, // default 20000
+  testsFinishTimeout: 180000, // default 20000
```

Comment:
why would this change require a longer timeout?

Reply:
See PR description. I think it's because the two are now running in parallel, so there must be some resource starvation that causes the test to take longer. It seemed to particularly affect Firefox.

Reply:
I bumped it even higher, to make it pass reliably. I think this is just a contention thing. But the good news is that overall test time seems lower: latest full run of the test step was 7m28s, compared to 9m58s on the most recent commit on main.